iscan/models.py

```diff
@@ -1082,6 +1084,25 @@ def run_enrichment(self):
             self.completed = False
             print(traceback.format_exc())
 
+class BackgroundTask(models.Model):
+    task_id = models.UUIDField(primary_key=True)
+    name = models.CharField(max_length=100)
+    corpus = models.ForeignKey(Corpus, on_delete=models.CASCADE)
+    running = models.BooleanField(default=True)
+    failed = models.BooleanField(default=False)
+
+    class Meta:
+        verbose_name_plural = 'Queries'
```

Comment:
Update verbose name or delete, I think it should pluralize it as "Background Tasks" by default

Reply:
whoops yeah